src/main/java/com/gmail/nossr50/util/blockmeta/chunkmeta/PrimitiveChunkStore.java

```diff
@@ -40,7 +43,7 @@
      * boolean isOk = ((code >>> offset) & 1) == 1;
      * </pre>
      */
-    public AtomicIntegerArray data;
+    public AtomicIntegerArray data; // should it private or protected?
```

Comment:
depends on whether or not you want classes in the same package to be able to access it